README.md

```diff
@@ -513,6 +515,26 @@ To add RAC to your application:
 To see a project already set up with RAC, check out [C-41][] or [GroceryList][],
 which are real iOS apps written using ReactiveObjC.
 
+#### Carthage
```

Comment:
Does the section need to be updated though? It seems too complicated if linking the library is enough (in regard to Carthage and Pods).